src/test/resources/hw1/AllTestsSuite.xml

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="AllTests suite">
+    <test name="Multiply test">
+        <classes>
+            <class name="hw1.CalculatorMultiplyTest"/>
+        </classes>
+    </test>
+
+    <test name="Divide test">
+        <classes>
+            <class name="hw1.CalculatorDivideTest"/>
+        </classes>
+    </test>
+
+    <test name="Sum test">
+        <classes>
+            <class name="hw1.CalculatorSumTest"/>
+        </classes>
+    </test>
+
+    <test name="Subtract test">
+        <classes>
+            <class name="hw1.CalculatorSubtractTest"/>
+        </classes>
+    </test>
```

Comment:
Why do you decide include all test classes into this suite? Do we have other way run all tests in the suite?

Reply:
We may use a package. Then all tests inside the package will run.